judges/github-events/github-events.rb

```diff
@@ -93,8 +93,7 @@ def self.fill_up_event(fact, json)
       end
 
     when 'PullRequestReviewEvent'
-      case json[:payload][:action]
-      when 'created'
+      if json[:payload][:pull_request][:user][:id].to_i != fact.who && json[:payload][:action] == 'created'
```

Comment:
@Suban05 to make this place in the script more compatible with other similar places I would rewrite it like this:

```
when 'PullRequestReviewEvent'
  case json[:payload][:action]
  when 'created'
    skip_event(json) if json[:payload][:pull_request][:user][:id].to_i == fact.who
    # here we process it 
```

Reply:
@yegor256 done